docs/enterprise-guide/dashboard-subscriptions.md

```diff
@@ -0,0 +1,28 @@
+
+## Customize filter values for each dashboard subscription
+
+This page covers the Enterprise Edition features for dashboard subscriptions. Our User's Guide can give you an introduction to [dashboard subscriptions](../users-guide/dashboard-subscriptions.md). 
```

Comment:
Can probably find a way to not repeat the phrase "dashboard subscriptions" at the end of both sentences.